imgee/models/stored_file.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 
 from coaster.sqlalchemy import BaseNameMixin, BaseScopedNameMixin
-import imgee
+from imgee import app, url_for
 from imgee.models import db
 from imgee.utils import newid, guess_extension
```

Comment:
Is there any advantage of using the relative import over absolute one? I usually feel a little confused whenever I face relative import. If the module name is present instead, it's clear on first glance where it's being imported from. 

And yeah, that's the same from flask.